libs/full/performance_counters/include/hpx/performance_counters/counters_fwd.hpp

```diff
@@ -28,31 +28,31 @@ namespace hpx { namespace performance_counters {
     inline std::string remove_counter_prefix(std::string const& counter);
 
     ///////////////////////////////////////////////////////////////////////////
-    enum counter_type
+    enum class counter_type
     {
         // \a counter_text shows a variable-length text string. It does not
         // deliver calculated values.
         //
         // Formula:  None
         // Average:  None
         // Type:     Text
-        counter_text,
+        counter_text = 0,
```

Comment:
I'd suggest renaming the enumeration values to reduce the naming noise, e.g. `counter_type::raw`, etc..

Reply:
done